prepare-docker-builds

```diff
@@ -16,12 +18,12 @@ fi
 echo '$BROCCOLI_SLEEP_LONG'="$BROCCOLI_SLEEP_LONG"
 
 if [ -z "$BROCCOLI_SLEEP_MEDIUM" ]; then
-  export BROCCOLI_SLEEP_MEDIUM=3
+  export BROCCOLI_SLEEP_MEDIUM=6
 fi
 echo '$BROCCOLI_SLEEP_MEDIUM'="$BROCCOLI_SLEEP_MEDIUM"
 
 if [ -z "$BROCCOLI_SLEEP_SHORT" ]; then
-  export BROCCOLI_SLEEP_SHORT=1
+  export BROCCOLI_SLEEP_SHORT=3
```

Comment:
Do we really need this here? We don't need to set these if they are set in the environment.

Reply:
agree